relay-dynamic-config/src/project.rs

```diff
@@ -54,6 +55,9 @@ pub struct ProjectConfig {
     /// Configuration for operation breakdown. Will be emitted only if present.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub breakdowns_v2: Option<BreakdownsConfig>,
+    /// Configuration for performance score calculations. Will be emitted only if present.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub performance_score_config: Option<PerformanceScoreConfig>,
```

Comment:
Just noticed that most keys in this config do not have a `_config` suffix, (e.g. `measurements: Option<MeasurementsConfig>`), should we remove it here as well?

Reply:
Yep, sure